items/weapons.xml

```diff
@@ -1859,10 +1859,10 @@
       </Weapon>
     </ItemComponent>
   </Item>
-  <Item id="crpg_crossbow_c" name="{=0e17RZrZ}Huntsman's Heavy Arbalest" body_name="bo_cross_bow_heavy" mesh="crossbow_c" culture="Culture.vlandia" weight="4.0" appearance="0.4" difficulty="20" Type="Crossbow" AmmoOffset="0.0, 0.02131, 0.24675" item_holsters="crossbow_back:bow_hip:mace_right_hip:bow_hip_2" holster_position_shift="0.02,0,-0.4">
+  <Item id="crpg_crossbow_c" name="{=0e17RZrZ}Heavy Hunting Arbalest" body_name="bo_cross_bow_heavy" mesh="crossbow_c" culture="Culture.vlandia" weight="4.0" appearance="0.4" difficulty="20" Type="Crossbow" AmmoOffset="0.0, 0.02131, 0.24675" item_holsters="crossbow_back:bow_hip:mace_right_hip:bow_hip_2" holster_position_shift="0.02,0,-0.4">
     <ItemComponent>
       <Weapon weapon_class="Crossbow" ammo_class="Bolt" missile_speed="120" accuracy="98" thrust_damage="50" thrust_speed="87" speed_rating="62" weapon_length="100" ammo_limit="1" thrust_damage_type="Pierce" item_usage="crossbow" reload_phase_count="2" physics_material="wood_weapon" center_of_mass="0,0,0.4" modifier_group="crossbow">
-        <WeaponFlags RangedWeapon="true" HasString="true" CantReloadOnHorseback="true" NotUsableWithOneHand="true" TwoHandIdleOnMount="true" />
+        <WeaponFlags RangedWeapon="true" HasString="true" CantReloadOnHorseback="true" CanPenetrateShield="true" NotUsableWithOneHand="true" TwoHandIdleOnMount="true" />

```

Comment:
Please discuss with me first major buff like this.

Reply:
Not really a buff to the new xbow, but an adjustment back to the value that both me and Queuex had discussed which was 55 reload speed, since going from a reload speed of 58 to 52 seemed pretty drastic.

Reply:
The CanPenetrateShield is only for the Heavy xbow, the lower and highest tier already have this property. (Heavy xbow only)

Reply:
And for the "CanPenetrateShield" affix on the rest of the heavy xbows was to make them more in line with the unique property of them being Heavy Xbows, though if that's too much of an overall buff for heavy xbows, I would say that the affix should be on both the Heavy Hunting Crossbow and the Knight's Heavy Arbalest as both are the high end xbows of their tiers.

Reply:
i agree on leaving that to the two higest tier
Please update your branch taking this into account